hack/gen-crd.sh

```diff
@@ -1,14 +1,12 @@
 #!/bin/bash
 set -euo pipefail
 
-REPO_ROOT=$(dirname "${BASH_SOURCE}")/..
-source $REPO_ROOT/hack/utils.sh
+controllerGen="go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go"
 
-# Generate manifests
-go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths=./... output:crd:artifacts:config=config/crd/bases
-sed -i '/^    controller-gen.kubebuilder.io\/version: (devel)/d' config/crd/bases/*
+# generate the CRD
+CRD_FILE=manifests/0000_30_cluster-baremetal-operator_00_metal3provisioning.crd.yaml
+CRD_OPTIONS="crd:trivialVersions=true,crdVersions=v1"
+$controllerGen $CRD_OPTIONS paths=./... output:stdout | sed '/^    controller-gen.kubebuilder.io\/version: (devel)/d' > $CRD_FILE
```

Comment:
Any particular reason for dropping `install_crd`?

Reply:
just seemed unnecessary to copy and have it in 2 palaces?
happy to revert this

Reply:
I think it's ok for now.
As a general comment, my personal feeling is that it would be better to keep the _raw_ generated manifests separated from the _rendered_ ones in the `/manifests` folder, anyhow we could revisit this strategy later I suppose.

Reply:
I have reverted it for now